index.html

```diff
@@ -0,0 +1,248 @@
+                        <li><a href="">Program</a></li>
+                        <li><a href="">Join</a></li>
+                        <li><a href="">Sponsor</a></li>
+                        <li><a href="">News</a></li>
+                        <li><button type="button">CC Campaign</button></li>
+                    </ul>
+                </div>
+            </nav>
+        </header>
+        <main>
+            <div class="head-bk flex">
+                <div class="title-top">
+                    <div class="title-1">
+                        <span>"Hello! Environmentalist"</span>
+                    </div>
+                    <div class="title-2">
+                        <span>PROPERTY & ENVIRONMENT<br> SUMMIT <br>2022</span>
+                    </div>
+                    <div class="title-2-1">
+                        <span>PROPERTY &<br> ENVIRONMENT SUMMIT <br>2022</span>
+                    </div>
+                </div>
+                <div class="title-3">
+                    <p class="p">Opportunities exist to develop conservation reserve networks
+                        in the western US that account for landscape connectivity and that benefit
+                        both private landowners and biodiversity conservation efforts through land trades and
+                        acquisitions.
+                    </p>
+                </div>
+                <div class="title-date">
+                    <h1 class="h1">2015.10.15(THU) ~ 16(FRI)</h1>
+                    <div>
+                        <span class="date-text">@ National Staduim of Uganda, Athlete Center point and more</span>
+                    </div>
+                </div>
+            </div>
+        </main>
+        <section class="main-container">
+            <div class="main-title flex">
+                <p class="main-t">Main Program</p>
+                <hr>
+            </div>
+            <div class="program-container">
+                <div class="program">
+                    <div class="program-theme">
+                        <img src="./images/program-01.png" class="prog-img flex" width="52" height="52" alt="">
+                        <h1 class="program-title">Lecture</h1>
+                    </div>
+                    <p class="program-text">
+                        Listen to the speakers from various countries about the messeges of sharing
+                        and opening.
+                    </p>
+
+                </div>
+
+                <div class="program">
+                    <div class="program-theme">
+                        <img src="./images/program-02.png" class="prog-img flex" width="52" height="52" alt="">
+                        <h1 class="program-title">CC Exhibition</h1>
+                    </div>
+                    <p class="program-text">
+                        Appreciate various creations applying CC license of artisits, organized
+                        from Art Center Nabi.
+                    </p>
```

Comment:
![image](https://user-images.githubusercontent.com/60612329/170844942-a98f025d-18a2-4522-b0a6-385bffe8bb60.png)

- [ ] in the mobile version of your main program section, your text is currently mixed up together 

- [ ]  on the program content please reduce the font size of the program title to fit the block content

Reply:
Required changes submitted